src/utils/isValidTokenId.ts

```diff
@@ -0,0 +1,23 @@
+/*-
+ *
+ * Hedera Token Holders List
+ *
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+export const isValidTokenId = (tokenId: string): boolean => {
+  const regex = /^0\.0\.\d*$/;
+  return regex.test(tokenId);
```

Comment:
https://docs.hedera.com/hedera/sdks-and-apis/hedera-api/basic-types/tokenid

using `0.0.` prefix will work for now, but a more general approach would be better

Reply:
Done